chroma_core/lib/job.py

```diff
@@ -200,7 +200,15 @@ def invoke_rust_agent(self, host, command, args={}):
     def invoke_rust_agent_expect_result(self, host, command, args={}):
         from chroma_core.services.job_scheduler.agent_rpc import AgentException
 
-        result = json.loads(self.invoke_rust_agent(host, command, args))
+        try:
```

Comment:
I think the only thing inside this try should be 

`result = self.invoke_rust_agent(host, command, args)`

Then the json parsing should be in a separate try afterwards.

Reply:
sounds good 👍